scripts/background.js

```diff
@@ -523,7 +526,11 @@ function setToolbarIcon(name) {
 
 function setToolbarState(tabId, name) {
   toolbarIconState[tabId] = name
```

Comment:
@cgorringe @anishsarangi I would like to know the best place for this line. Should it be placed inside the **if ( )** statement, Line No. 530?

 `if (tabs[0].id === tabId) {
      setToolbarIcon(name)
    }`

I couldn't find any differences while testing the extension.